code/modules/hydroponics/grown/ambrosia.dm

```diff
@@ -61,7 +61,7 @@
 	plantname = "Ambrosia Gaia"
 	product = /obj/item/food/grown/ambrosia/gaia
 	mutatelist = list(/obj/item/seeds/ambrosia/deus)
-	reagents_add = list(/datum/reagent/consumable/nutriment = 0.06, /datum/reagent/consumable/nutriment/vitamin = 0.05)
+	reagents_add = list(/datum/reagent/consumable/nutriment = 0.06, /datum/reagent/consumable/nutriment/vitamin = 0.05, /datum/reagent/medicine/earthsblood = 0.05)
```

Comment:
This is the only thing im a bit iffy about but everything else looks good.

Reply:
I figured it wouldn't be too strong as 1. It does brain damage which you need another person so you can deal with it, 2. It pacifies when while in the person's body, which means no suppressing abnormalities when healing up with it, and 3. Usually teamwork like this isn't used. I can't count the number of times agents don't use healing items that clerks set up for them. But if it is that strong, I could always adjust the values if needed.